homeassistant/components/transmission/strings.json

```
@@ -13,12 +13,11 @@
       }
     },
     "error": {
-      "name_exists": "Name already exists",
-      "wrong_credentials": "Wrong username or password",
-      "cannot_connect": "Unable to Connect to host"
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]"
     },
     "abort": {
-      "already_configured": "Host is already configured."
+      "already_configured_device": "[%key:common::config_flow::abort::already_configured_device%]"
```

We should not rename this key.
```suggestion
      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
```